### ISSUE TITLE
Forth solution is displayed incorrectly in solutions page

### ISSUE THREAD

**Issue by Voileexperiments** [Codewars/codewars.com#1942]
In the Forth version of [this kata](https://www.codewars.com/kata/product-of-consecutive-fib-numbers/forth) I submitted the following code:

```forth
: product-fib { p } 0 1 begin 2dup * p < while tuck + repeat 2dup * p = ;
```

But it's saved as [this](https://www.codewars.com/kata/reviews/5d7b41d801af060001a0ff20/groups/5d7e2caea4b23f0001ccdfcb):

```forth
: product-fib { p } 0 1 begin dup * p < while tuck + repeat dup * p = ;
```

Which is absurd, since this saved version doesn't even work.

**Comment by kazk** [comment]
Probably something similar to #1436. Highlightjs doesn't have Forth support. I'd like to eventually remove it and use CodeMirror to do highlights.

**Comment by kazk** [comment]
Forking it will show the submitted code so the stored data is correct. I'm pretty sure it's highlightjs issue and same as #1436.

**Comment by kazk** [comment]
Fixed by replacing with CodeMirror (#1778)
